### DISCUSSION TITLE
[FEATURE]: Add sales volume info to getListingsByListingIds

### DISCUSSION THREAD

**Discussion by nctbrtc** [etsy/open-api#721] [Ideas]
### Is there an existing ticket for this?

- [X] I have searched the existing tickets

### Is your feature request related to a problem? Please describe

I want to know how many times a listing made a sale 

### Describe the solution you'd like

when we use getListingsByListingIds, there can be **_sales_** value as well that shows how many times it's sold

**Comment by mdriess** [comment]
@nctbrtc you might want to add a name to the feature request so it's more clear?

**Comment by etsyachristensen** [comment]
@nctbrtc I can't say whether this is something we can do. You should be able to gather this information via the Receipt endpoints. I will add it to our requested features list.

**Comment by nctbrtc** [comment]
@etsyachristensen  thank you very much for fixing the name and for your response. 
and thank you for pointing to Receipt endpoints. I know we can get counts by **getShopReceiptTransactionsByListing**

is there any endpoint or method to get all sales within a range (date) with listing IDs in one request?

with one request, I want to know which listings got a sale for the requested time frame, like yesterday

**Comment by etsyachristensen** [comment]
Unfortunately no not at this time.

**Comment by etsyachristensen** [comment]
We are moving this to an Idea Discussion to continue tracking when we close the Issues board down.
